saveJsonFile.py

```diff
@@ -0,0 +1,131 @@
+# -*- coding: utf-8 -*-
```

Comment:
This file looks like it has the main method and does the yelp API look ups. So it's a standalone script that does the look up using yelp api. Could we rename this file to yelpApiCrawl.py or something more meaningful?

Reply:
OK! I will change name that is "yelpApiCraw.py", and I will change website_search.py to yelpHtmlCraw.py"!!